### ISSUE TITLE
Function checking fails when missing variables / functions

### ISSUE THREAD

**Issue by djw8605** [osg-bosco/GridR#1]
The function checking fails when checking for missing variables / functions in the functions that are sent to the remote clusters.

For example, this set of functions will not work:

```
> a <- function(s){return (b(s)*2}
> b <- function(s){return (s*3)}
> grid.apply("x", a, 10)
```

The above will not work because the function `a` cannot find function `b` when dependency resolving.

**Comment by djw8605** [comment]
Fixed in fd0dcc8.

**Comment by djw8605** [comment]
This problem has been fixed.  The above example in the ticket can be send to the remote cluster now.
